programmer/tinyprog/__main__.py

```diff
@@ -292,180 +292,184 @@ def parse_int(str_value):
     tinyprog.use_libusb = args.libusb
     tinyprog.use_pyserial = args.pyserial
 
-    active_boards = get_ports(device) + get_ports("1209:2100")
+    try:
+        active_boards = get_ports(device) + get_ports("1209:2100")
 
-    if args.meta:
-        meta = []
-        for port in active_boards:
-            with port:
-                p = TinyProg(port)
-                m = p.meta.root
-                m["port"] = str(port)
-                meta.append(m)
-        print(json.dumps(meta, indent=2))
-        sys.exit(0)
+        if args.meta:
+            meta = []
+            for port in active_boards:
+                with port:
+                    p = TinyProg(port)
+                    m = p.meta.root
+                    m["port"] = str(port)
+                    meta.append(m)
+            print(json.dumps(meta, indent=2))
+            sys.exit(0)
 
-    print("")
-    print("    TinyProg CLI")
-    print("    ------------")
+        print("")
+        print("    TinyProg CLI")
+        print("    ------------")
 
-    print("    Using device id {}".format(device))
+        print("    Using device id {}".format(device))
 
-    # find port to use
-    active_port = None
-    if args.com is not None:
-        active_port = tinyprog.SerialPort(args.com)
+        # find port to use
+        active_port = None
+        if args.com is not None:
+            active_port = tinyprog.SerialPort(args.com)
 
-    elif args.id is not None:
-        active_port = get_port_by_uuid(device, args.id)
+        elif args.id is not None:
+            active_port = get_port_by_uuid(device, args.id)
 
-    elif not active_boards:
-        print("""\
-    No port was specified and no active bootloaders found.
-    Activate bootloader by pressing the reset button.
-""")
-        sys.exit(1)
+        elif not active_boards:
+            print("""\
```

Comment:
Don't indent the multi-quote strings please.

Reply:
Thanks for the hint.
I reverted the changes to the multi-line-strings and pushed a new version.